applications/google_rise_2016.md

```diff
@@ -311,27 +332,32 @@ curriculum, which will be led by the curriculum lead we hire.
 
 **Who will be leading this project? Include the title and relevant experience.**
 
-Zachary Latta, Executive Director of Hack Club.
+Zach Latta, Executive Director of Hack Club.
 
-A former club leader himself, Latta began Hack Club after he realized how much
-of an impact finding peers and forming a coding community made on him in terms
-of confidence, mentorship, and impetus to improve. Having run Hack Club for over
-a year now, Latta has experience writing curriculum and running workshops.
+Zach started Hack Club after he started a coding club at his high school and
+wanted to be part of something larger than his school. Before working on Hack
+Club full-time, Zach built the backend architecture for Yo -- an app with more
+than 3 million users. Prior to that, he was a programmer of Football Heroes, a
+game with over a million users.
```

Comment:
Wait, i think you should emphasize how you LED the development of backend for Yo, and then how you ... did something to do with the million users in Football Heroes. Or maybe say that you realized how great it was to have an impact on people after building Football Heroes. Something slightly more relevant (at least in phrasing). The numbers are a good indicator of impact, but it's be good to make it a little more obvious how it reflects back onto you/your experience.